apisix/discovery/consul/init.lua

```diff
@@ -161,12 +175,11 @@ local function show_dump_file()
     if not data then
         return 503, err
     end
-
```

Comment:
Why do you remove this blank line?

Reply:
just re-fomat the code

Reply:
I think we can keep this, like other place in you file

Reply:
@Fabriceli

Reply:
done